tools/clang/test/HLSLFileCheck/passes/llvm/globalopt/alias-used.ll

```diff
@@ -0,0 +1,49 @@
+; RUN: %opt %s -globalopt -S | FileCheck %s
+
+; Not in DXC? ; XXCHECK: @main
```

Comment:
What is this mean? I didn't notice that check in the original test.

Reply:
Argh, mistake of mine when getting this test to run. One of the other .ll files in this area uses @main. I'll delete it